File: BackEndApi/Services/DALModels/Tickets.cs
using BackEndApi.Services.DALModels;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace BackEndApi.Models.Ticket
{
    public class Tickets
    {
        [Key]
        public int TicketID { get; set; }
        public string SubmitterName { get; set; }
        public string SubmitterEmail { get; set; }
        public string SubjectLine { get; set; }
        public string Problem { get; set; }
        public DateTime SubmittedTime { get; set; } = DateTime.UtcNow;

        public string Solution { get; set; }
        public int Priority { get; set; }
        public string AssignedTo { get; set; }
        public string Status { get; set; }
        public DateTime ResolvedTime { get; set; }

        //[ForeignKey("Users")]
        //public int ID { get; set; }
        public IEnumerable<Users> Users { get; set; }

    }
}

File: Help-Desk-Angular/src/app/models/Tickets.ts
// export interface Tickets{
//     ticketId: number;
//     submitterName: string;
//     submitterEmail: string;
//     subject: string;
//     problem: string;
//     solution: string;
// }

export interface Tickets {
    ticketID: number;
    submitterName: string;
    submitterEmail: string;
    subjectLine: string;
    problem: string;
    submittedTime: string;
    solution: string;
    priority: number;
    assignedTo: string;
    status: string;
    resolvedTime: string;
    users: any;
  }



export type PostTicket = Omit<Tickets, "ticketId">;